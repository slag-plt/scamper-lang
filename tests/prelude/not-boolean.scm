(not #t)
(not #f)
(not 1)
(not 0)
(not 2)
(boolean? #t)
(boolean? #f)
(boolean? 0)
(boolean? 2)
(boolean? 1)