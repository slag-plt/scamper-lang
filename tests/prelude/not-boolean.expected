#f
#t
:1:2: Runtime error:
    not expected an boolean? in position 1 but a lit was given
    In program: (not 1)
:1:3: Runtime error:
    not expected an boolean? in position 1 but a lit was given
    In program: (not 0)
:1:4: Runtime error:
    not expected an boolean? in position 1 but a lit was given
    In program: (not 2)
#t
#t
#f
#f
#f
